pages/project/read/views/index.jsx

```diff
@@ -66,6 +66,10 @@ export default function ProjectLandingPage() {
     delete sections.downloads;
   }
 
+  if (asruUser && asruInspector) {
+    delete sections.manage;
+  }
```

Comment:
This feels like the wrong way round to approach this. For example in the case where a person has inspector and licensing roles it should remain visible.

Ensure that the tab _does_ appear for the people who _should_ see it, and don't include inspector on that list.